src/main/java/org/stellar/sdk/requests/AssetsRequestBuilder.java

```diff
@@ -39,4 +39,22 @@ public static Page<AssetResponse> execute(OkHttpClient httpClient, HttpUrl uri)
   public Page<AssetResponse> execute() throws IOException, TooManyRequestsException {
     return this.execute(this.httpClient, this.buildUri());
   }
+
+  @Override
```

Comment:
looks good, can you add a test case in `AssetsRequestBuilderTest` to assert these new setters result in expected output from`AssetsRequestBuilder.buildUri()`  ?

Reply:
We now have such a test. https://github.com/stellar/java-stellar-sdk/blob/305974255130a2ebe42aeb289ed7040de8d75892/src/test/java/org/stellar/sdk/requests/AssetsRequestBuilderTest.java#L18-L20